sky/backends/backend_utils.py

```diff
@@ -1124,7 +1126,15 @@ def write_cluster_config(
         # pylint: disable=import-outside-toplevel
         from sky.skylet.providers.gcp import config as gcp_config
```

Comment:
Maybe add a comment here that: `We do not import the module under sky.skylet.providers globally as we need to avoid importing ray module, which is not in the dependency of AWS support`.

Reply:
Good call, added!